### ISSUE TITLE
CameraController(frontCamera, ResolutionPreset.low) params are limit

### ISSUE THREAD

**Issue by wcyDreamer** [flutter/flutter#45665]
there are only 5 enum selections.and we can not customize the size of picture which i take.
i want to get a pic with low's Rotate and with the quality of veryhigh.and i try everything.
why the Rotate of low is different with the Rotate of veryhigh.

**Comment by wcyDreamer** [comment]
for example if i want to take a pic with 960*1280,what i can do.

**Comment by LTY2009** [comment]
I also miss this question.how about setting a fixed aspect ratio firstly and then chosing the quality.

**Comment by iapicca** [comment]
Hi @LTY2009 
I think you are making a good point,
I feel that there's no harm in widening the topic a bit,
updating the title.
Thank you

**Comment by wcyDreamer** [comment]
anybody else ?

**Comment by xyzira** [comment]
I'm annoyed by the fixed aspect ratio in the resolution preset. Ultra high, hight and Max all have 16:9 ratio, that mean the cameras with 4:3 sensor have areas cut off

**Comment by lahdekorpi** [comment]
This is a major oversight in the plugin. With low being, the only one with a 4:3 aspect ratio means that all the other ones are cropping into the sensor. If you want to have the widest field of view, you would have to either use the low resolution or write a custom native camera plugin?

Are there any workarounds or forked versions that solve this?

**Comment by G2G2G2G** [comment]
I need it as well. How can you not do something this basic?

**Comment by migalv** [comment]
Yeah same issue here. There is no way to set custom aspect ratios.

**Comment by G2G2G2G** [comment]
I still need it as well. Marking my on topic comment off topic won't excuse your inability to do something this basic. Just add aspect ratios lmao.

**Comment by Shawn-sudo** [comment]
@G2G2G2G Do you not understand why Flutter is an open source project? It's not for you to complain and blame others, but for us (including you!) to help and participate the project to grow. If this is really that "basic" for you, I'd suggest you to create a new branch with your codes committed and open a new pull request.

**Comment by G2G2G2G** [comment]
stop protecting google employees.

**Comment by 18manfish** [comment]
@Shawn-sudo stop sucking googles dick, they are not going to hire you

**Comment by darshankawar** [comment]
Please be aware of the wordings used in the comments and I strongly recommend to go through [code of conduct](https://docs.github.com/en/site-policy/github-terms/github-event-code-of-conduct#code-of-conduct) to adhere to while commenting.

**Comment by 18manfish** [comment]
kys

**Comment by G2G2G2G** [comment]
Stop the spam @Shawn-sudo  and @darshankawar stay on topic. If you stop wasting so much time this issue could already be resolved.

**Comment by panzikeji** [comment]
Do you have a good solution to this problem now

**Comment by lahdekorpi** [comment]
> Do you have a good solution to this problem now

Yes, https://github.com/Apparence-io/CamerAwesome

**Comment by ioridev** [comment]
I think CamerAwesome is just cropping an image that was cropped to 16:9 to 4:3 again. I need an option to use the full sensor.
